docs/src/rules/no-undefined.md

```diff
@@ -57,6 +57,9 @@ if (foo === undefined) {
 function foo(undefined) {
     // ...
 }
+
+// Skips the first parameter
```

Comment:
This is a bit confusing, what do you mean by "skips"? I think this comment is making it more confusing and we can just remove it.

This is in the "incorrect" section, so this would be flagged as incorrect by `no-undefined`

Reply:
Would the original comments (plus spelling fixes) be better?

Reply:
I think it would be best to just remove the comments as the intent seems clear from the code.